src/content/release-notes.mdx

```diff
@@ -5,6 +5,39 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.23.0
+
+12 August 2024
+
+This versions is compatible with Kubernetes versions 1.26 to 1.30
+
+### New Features
+
+- Added a [feature flag to enable/disable autowake](self-hosted/helm-configuration.mdx#autowake) for Namespaces on incoming request  <!-- 8379 -->
+- Support for GitLab Subgroups and Groups with nested Subgroups <!-- 8398, 8393, 8394 -->
+- Added support for Kubernetes 1.30  <!-- 8421 -->
+
+### Improvements
+
+- Prevent the usage of reserved variable names  <!-- 8272 -->
+- Fix for git-urls CVE-2023-46402  <!-- 8300 -->
+- Updated registry version to 2.8.2-75b5eee  <!-- 8368 -->
+- Reconcile selected resource state consistently with URL  <!-- 8363 -->
+- Redesigned the `Setting` Dashboard component to show status (e.g. Garbage Collector installed/not installed)  <!-- 8380 -->
+
+### Bug Fixes
+
+- For administrators, the Dashboard now shows the admin alert banner when looking at other users' empty Namespaces and destroy logs  <!-- 8293 -->
+- Disabled the delete button for any Personal Namespace  <!-- 8295 -->
+- Inject the okteto-cluster-admin service account into installer jobs and grant registry access to all admins  <!-- 8299 -->
```

Comment:
The same than previous 2 points. This is a fix for a regression introduced in a change that never went live